interp/src/flatten/structures/environment/env.rs

```diff
@@ -248,6 +249,63 @@ impl<C: AsRef<Context> + Clone> Environment<C> {
         self.ctx.as_ref()
     }
 
+    pub fn iter_cells(
```

Comment:
Add a docstring saying what this return type is (full name of cell and the names of attached ports)